rs/sns/governance/src/governance.rs

```diff
@@ -151,6 +152,9 @@ pub fn log_prefix() -> String {
 /// The static MEMO used when calculating the SNS Treasury subaccount.
 pub const TREASURY_SUBACCOUNT_NONCE: u64 = 0;
 
+// How frequently the canister should attempt to refresh the cached_upgrade_steps
+pub const UPGRADE_STEPS_INTERVAL_REFRESH_BACKOFF_SECONDS: u64 = 5 * 60; // 5 minutes
```

Comment:
This seems more frequent that is needed.  For hotfixes, there'd be a push notification, correct?  So this could easily be more like an hour without it making much difference.

Reply:
Done